mlflow/azureml/__init__.py

```diff
@@ -311,6 +469,32 @@ def _get_mlflow_azure_resource_name():
             max_length=(azureml_max_resource_length - len(resource_prefix)))
     return resource_prefix + unique_id
 
+def _get_mlflow_azure_name_from_run(run_id):
+    """
+    :return: A unique name for an Azure resource indicating that the resource was created by
+             MLflow
+    """
+    azureml_max_resource_length = 32
+    resource_prefix = "mlflow-run-"
+    suffix = run_id[:azureml_max_resource_length-len(resource_prefix)]
+    return resource_prefix + suffix
+  
+
+def _create_mlflow_wheel(mlflow_dir, out_dir):
+    """
+    Create the wheel of MLFlow by using setup.py bdist_wheel in the outdir.
+
+    :param model_path: The absolute path to the mlflow.
+    :param out_dir: The absolute path to the outdir.
+    :return: The absolute path to the wheel.
+    """
+    out_path = Path(out_dir).resolve()
+    process = subprocess.run([sys.executable, "setup.py", "bdist_wheel", "-d", out_path], cwd=mlflow_dir, check=True)
+    files = list(out_path.glob("./*.whl"))
+    if len(files) != 1:
```

Comment:
Nit: What happens if `len(files) > 1`? The error message doesn't seem to apply in this case (we found the wheel, but there were multiple such wheels...)